request_option.go

```diff
@@ -15,60 +15,81 @@ func WithKeyValue(k string, v interface{}) RequestOption {
 func WithTimestamp(ts time.Time) RequestOption {
 	return WithKeyValue("timestamp", ts.Unix())
 }
+
```

Comment:
Why there are not commentaries? This would be nice if it is provided. At least for some of them, but I suggest you to add text above each of the functions

Reply:
The problem is that these may be used in different contexts (for different API calls) and may have different connotation. I have some ideas how to improve it but better discuss it in person.